webapp/sass/responsive/_mobile.scss

```diff
@@ -1276,6 +1276,24 @@
                     height: auto;
                 }
             }
+            .post__header {
+                .col__name {
+                    .user-popover {
+                        max-width: 80px;
```

Comment:
@asaadmahmood Thanks for your feedback. Unfortunately, I don't have the bandwidth at this time to continue with this PR. There are complications preventing the additional 10-20px that visually seem available, and to fix this will require changes to multiple parts of the codebase. Please feel free @asaadmahmood or anyone to pick this up if you'd like.

Cheers.